test/OpenTelemetry.Tests/Implementation/Internal/TestHttpServer.cs

```diff
@@ -57,23 +56,23 @@ public RunningServer(Action<HttpListenerContext> action, string host, int port)
                             {
                                 break;
                             }
-                            Assert.True(false, ex.ToString());
```

Comment:
Small change here. Benchmarks doesn't have Xunit reference so I decided to make it a throw instead. No code reliant on this Assert behavior that I could tell, it's more of an alert just in case something goes haywire.